Makefile

```diff
@@ -11,7 +11,7 @@
 ##
 
 # App name used to created our Docker image. This name is important in the context of the AWS docker repository.
-APP_NAME = parliamentukvarnish
+APP_NAME = parliamentukrouting
```

Comment:
This change is for consistency: the GitHub project is already called "routing" so we should stick with that.